content/docs/howto/java.md

```diff
@@ -400,15 +400,18 @@ The following table documents the versions available.
 
 | GraalVM Version | Java Native Image Buildpack Version |
 | --------------- | ----------------------------------- |
+| 22.0            | 7.27.0                              |
```

Comment:
```suggestion
| 22.2           | 7.26.1                              |
```
Should this be 22.2? Also the version releasing today was a patch so the latest will be 7.26.1